### ISSUE TITLE
Recording performance not working

### ISSUE THREAD

**Issue by onlineversioncontrolsystem** [chronopolis5k/Grainstation-C#5]
Record Arm doesn't seem to do anything for me; no blinking and no recorded output in the folders

Video: https://www.dropbox.com/s/v8gzyxugyay8htr/Grainstation%20C%20Record%20Arm%20issue.mp4?dl=0

- Have downloaded fresh repository
- Checked to see that the templates were loaded properly on Launchcontrol XL mk2
- Confirmed in Ableton that in user templates the Record Arm button toggles a C7 note

Other controls seem to be working as intended, although possibly more laggy that I would expect from midi control but perhaps this just how it is

**Comment by micah-frank-studio** [comment]
Hi - Sorry for the delay here. I will look into this next week.

**Comment by micah-frank-studio** [comment]
Hi - really sorry for the incredible delay here. What do you have for your midi I/O settings? Can you post a screenshot?

**Comment by onlineversioncontrolsystem** [comment]
Screenshot included of my settings, where you can also see the error message that appears when the record button is pressed. Second screenshot of the other midi devices that are available.

![Grainstation-Screenshot01](https://user-images.githubusercontent.com/741213/69482563-fb82a780-0e14-11ea-8598-8201e8ce4ef2.png)
![Grainstation-Screenshot02](https://user-images.githubusercontent.com/741213/69482564-fcb3d480-0e14-11ea-97bd-cecc16117634.png)

**Comment by micah-frank-studio** [comment]
Hi,
make sure you have a "renders" folder there in the Grainstation-C-master
folder.

**Comment by onlineversioncontrolsystem** [comment]
Yes, I have the renders folder, the folder structure is straight out of the github zip. The folder is in my dropbox but I got exactly the same results when I ran it from my desktop

![Untitled](https://user-images.githubusercontent.com/741213/69633503-40425480-1048-11ea-8518-08f8768209d5.png)

**Comment by micah-frank-studio** [comment]
on line 862 try changing

`fout Sfilename, 24, allL, allR`

to

`fout "file.wav", 24, all, allR`

and see if it renders a file right in the main repo directory. I think it
must be a permissions issue so it cannot write to renders/

**Comment by onlineversioncontrolsystem** [comment]
I still get this error 

`INIT ERROR in instr 100 line 880: error opening sound file 'b_format/Mon Dec  2 20:36:37 2019
.wav'
 from file C:\Users\josha\Dropbox\Grainstation C\Grainstation-C-master\Grainstation-C.csd (1)
	fout.A	Sfilename	24	abform	
Closing file 'file.wav'...
		   T  4.087 - note deleted.  i100 had 1 init errors`

But it does create an empty "file.wav" in the root folder.

(I changed all to allL form your code to make it work)

**Comment by micah-frank-studio** [comment]
> (I changed all to allL form your code to make it work)
Ahh yeah, typo on my end. Sorry.
